web/src/components/Form1.js

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+const Form1 = ({ handleSubmit, pristine, reset, submitting }) => (
+  <form onSubmit={handleSubmit}>
+    <div>
+      <label htmlFor="firstName">First Name</label>
+      <div>
+        <Field
+          id="firstName"
+          name="firstName"
+          component="input"
+          type="text"
+          placeholder="First Name"
+        />
+      </div>
+    </div>
+    <div>
+      <label htmlFor="firstName">Last Name</label>
```

Comment:
I think this should be `htmlFor="lastName"`?  :)

Reply:
yes, thx! (and i feel like that duplicate label keys should be a linting check 🙃)